Add initial version of 3scale alert SOPs included on 3scale v2.9 (#9)
3scale/3scale-Operations: sops/alerts/prometheus_job_down.adoc

:toc:
:toc-placement!:

= ThreescalePrometheusJobDown

toc::[]

== Description

* This alert willl trigger when a prometheus Job is Down, because `ServiceMonitor`/`PodMonitor` might have issues to scrape the application `Service`/`Pods`

> * It is a **really important alert**, because if you have alerts based on applicatin metrics that cannot be obtained because its associated prometheus job is failing, then you lose visibility of the application

Reviewer: ```suggestion
* This alert is important because if you have alerts based on application metrics that cannot be obtained because its associated prometheus job is failing you will lose visibility of the application
```

== Troubleshooting

> * Check if scrapped application (or app metrics endpoint) might be failing, so scrape can not be done

Reviewer: ```suggestion
* Check if the scrapped application (or app metrics endpoint) might be failing and the scrape can not be done
```

> * Check if `ServiceMonitor`/`PodMonitor` might be bad configured (pointing to incorrect label...)

Reviewer: ```suggestion
* Check if `ServiceMonitor`/`PodMonitor` might be misconfigured pointing to incorrect label, port, path...
```
